.gitignore

```diff
@@ -1 +1,3 @@
-.vscode/*
\ No newline at end of file
+.vscode/*
+hack/cloud-init.sh
+hack/test_ssh*
```

Comment:
these scripts are in gitignore, but are being used in `hack/check-uptime.sh` and in `hack/upgrade-cnv.sh`

Reply:
`hack/test_ssh*` are the SSH key pair files generated on the fly in `hack/upgrade-cnv.sh`,
`hack/cloud-init.sh` is also rendered on the fly by `hack/upgrade-cnv.sh` to be injected on the VM because Cirros doesn't support cloud-config (yaml based) cloud-init files but only sh scripts.
Since these files contain SSH keys generated on the fly, we don't want to commit them by mistake in the repo.